testing/ceph/APKBUILD

```diff
@@ -178,44 +191,18 @@ base() {
 		xfsprogs
 	"
 
-	mkdir -p "$subpkgdir"$_docdir/ceph
-	mv "$pkgdir"$_docdir/ceph/sample.ceph.conf \
-		"$subpkgdir"$_docdir/ceph/sample.ceph.conf
-	mv "$pkgdir"$_docdir/ceph/sample.fetch_config \
-		"$subpkgdir"$_docdir/ceph/sample.fetch_config
-
-	mkdir -p "$subpkgdir"$_bindir
-	for file in crushtool monmaptool osdmaptool ceph-run ceph-detect-init; do
-		mv "$pkgdir"$_bindir/$file "$subpkgdir"$_bindir
-	done
-
-	mkdir -p "$subpkgdir"$_sbindir
-	mv "$pkgdir"$_sbindir/ceph-create-keys "$subpkgdir"$_sbindir/ceph-create-keys
-	mv "$pkgdir"$_sbindir/mount.ceph "$subpkgdir"$_sbindir/mount.ceph
-
-	mkdir -p "$subpkgdir"$_libexecdir/ceph
-	mv "$pkgdir"$_libexecdir/ceph/ceph_common.sh "$subpkgdir"$_libexecdir/ceph
-
-	mkdir -p "$subpkgdir"$_libdir/rados-classes
-	mv "$pkgdir"$_libdir/rados-classes/* "$subpkgdir"$_libdir/rados-classes
-	mkdir -p "$subpkgdir"$_libdir/ceph/erasure-code
-	mv "$pkgdir"$_libdir/ceph/erasure-code/libec_*.so* \
-		"$subpkgdir"$_libdir/ceph/erasure-code
-	mkdir -p "$subpkgdir"$_libdir/ceph/compressor
-	mv "$pkgdir"$_libdir/ceph/compressor/libceph_*.so* \
-		"$subpkgdir"$_libdir/ceph/compressor
-
-	mkdir -p "$subpkgdir"$_sysconfdir/logrotate.d
-	mv "$pkgdir"$_sysconfdir/logrotate.d/ceph \
-		"$subpkgdir"$_sysconfdir/logrotate.d/ceph
-
-	mkdir -p "$subpkgdir"$_python_sitelib
-	mv "$pkgdir"$_python_sitelib/ceph_detect_init* "$subpkgdir"$_python_sitelib
-	mv "$pkgdir"$_python_sitelib/ceph_disk* "$subpkgdir"$_python_sitelib
-
+	_pkg $_docdir/ceph sample.ceph.conf sample.fetch_config
```

Comment:
Unfortunately APKBUILD does not run with `set -e` (yet, it’s on our TODO list to change it), so every statement that may fail, but should not fail, and is not the last statement of the function, should end with `|| return 1`.